spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskExecutionService.java

```diff
@@ -315,28 +314,14 @@ public long executeTask(String taskName, Map<String, String> taskDeploymentPrope
 		TaskExecutionInformation taskExecutionInformation = findOrCreateTaskExecutionInformation(taskName,
 				taskDeploymentProperties, launcher.getType(), previousTaskDeploymentProperties);
 
-		// pre prosess command-line args
-		// moving things like app.<label> = arg
-		// into deployment properties if ctr and removing
-		// prefix if simple task.
 		if (taskExecutionInformation.isComposed()) {
-			List<String> composedTaskArguments = new ArrayList<>();
-			commandLineArgs.forEach(arg -> {
-				if (arg.startsWith("app.")) {
-					String[] split = arg.split("=", 2);
-					if (split.length == 2) {
-						composedTaskArguments.add("--composed-task-app-arguments." + Base64Utils.encode(split[0]) + "=" + split[1]);
-					}
-					else {
-						composedTaskArguments.add("--composed-task-app-arguments." + arg);
-					}
-				}
-				else {
-					composedTaskArguments.add(arg);
-				}
-			});
-			logger.info("composedTaskArguments {}", StringUtils.collectionToCommaDelimitedString(composedTaskArguments));
-			commandLineArgs = composedTaskArguments;
+			// pre process command-line args
```

Comment:
**[TinyNit]** I know this was like this before, but could we merge this comment onto fewer lines ? :)

Reply:
Agreed